src/test/java/com/olegdvd/temperature/service/SensorRequestServiceTest.java

```diff
@@ -1,20 +1,15 @@
 package com.olegdvd.temperature.service;
 
 import com.olegdvd.temperature.domain.GatheredSensorData;
-import com.olegdvd.temperature.domain.SensorData;
 import com.olegdvd.temperature.domain.SensorInfo;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.orm.jpa.AutoConfigureDataJpa;
-import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.Optional;
-
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 
```

Comment:
For unit test any external dependencies should be mocked. In such condition it is raser end-to-end test, and should be @Ignore-d in unit tests

Reply:
Removed test that process with real dependancy.